### ISSUE TITLE
Code is generated for empty XPath results

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Kramermp/htmlunitscripter#12]
```
Some html elements, such as the HtmlParagraphElement, get processed by 
HtmlUnitScripter even though there is nothing to click on. The code attempts to 
run an XPath query, but the query returns empty.

HtmlUnitScripter needs to be modified to not output code when the XPath query 
returns nothing.

```

Original issue reported on code.google.com by `gross...@gmail.com` on 10 Nov 2010 at 4:49

**Comment by GoogleCodeExporter** [comment]
```
Issue has been fixed in the 1.1.4 release.
```

Original comment by `gross...@gmail.com` on 10 Nov 2010 at 5:29
- Added labels: **Milestone-Release1.1.4**

**Comment by GoogleCodeExporter** [comment]
Original comment by `gross...@gmail.com` on 10 Nov 2010 at 5:30
- Changed state: **Verified**
